x/incentive/types/keys.go

```diff
@@ -24,11 +24,11 @@ const (
 )
 
 var (
-	ParamsKey                                  = []byte{0x01}              // key prefix for the parameters
-	BTCStakingGaugeKey                         = []byte{0x02}              // key prefix for BTC staking gauge at each height
-	DelegatorWithdrawAddrPrefix                = []byte{0x03}              // key for delegator withdraw address
-	RewardGaugeKey                             = []byte{0x04}              // key prefix for reward gauge for a given stakeholder in a given type
-	RefundableMsgKeySetPrefix                  = collections.NewPrefix(5)  // key prefix for refundable msg key set
+	ParamsKey                   = []byte{0x01} // key prefix for the parameters
+	BTCStakingGaugeKey          = []byte{0x02} // key prefix for BTC staking gauge at each height
+	DelegatorWithdrawAddrPrefix = []byte{0x03} // key for delegator withdraw address
+	RewardGaugeKey              = []byte{0x04} // key prefix for reward gauge for a given stakeholder in a given type
+	// RefundableMsgKeySetPrefix = collections.NewPrefix(5) is deprecated since we don't use refundable msg key set anymore
```

Comment:
I believe we should add a migration to make sure to remove all data stored in this key prefix (if anything remains).
Wdyt @KonradStaniec @RafilxTenfen ?